### PR TITLE
Modifies LectureServlet to respond with one lecture 

### PR DESCRIPTION
**Pull request by jaysaleh**
This pr modifies the `LectureServlet` to respond with one lecture.

### CODE REVIEW

File: src/main/webapp/view/view.js [hunk 1]
```diff
@@ -31,6 +34,14 @@ async function initialize() {
   window.loadTranscript(window.location.search);
 }
 
+/** Returns lecture in database associated with {@code window.LECTURE_ID}. */
+async function getLecture() {
+  const url = new URL('/lecture', window.location.origin);
```
Comment (n-wach) [inline src/main/webapp/view/view.js:39]: `ENDPOINT_LECTURE` const + add to comment

File: src/test/java/com/googleinterns/zoomtube/servlets/LectureServletTest.java [hunk 2]
```diff
@@ -99,6 +106,37 @@ public void doPost_urlNotInDatabase_shouldAddToDatabaseAndReturnRedirect() throw
     verify(response).sendRedirect("/view/?id=1&video-id=wXhTHyIgQ_U");
   }
 
+  @Test
+  public void doGet_lectureInDatabase_shouldWriteLecture() throws IOException {
+    Entity lectureEntity = LectureUtil.createEntity(/* lectureName= */ "", TEST_LINK, TEST_ID);
```
Comment (n-wach) [inline src/test/java/com/googleinterns/zoomtube/servlets/LectureServletTest.java:111]: Can you test with a non-empty string for name?

File: src/main/webapp/view/view.js [hunk 3]
```diff
@@ -19,7 +19,10 @@ const PARAM_VIDEO_ID = 'video-id';
 window.LECTURE_ID = getLectureId();
 window.VIDEO_ID = getVideoId();
 
-initialize();
+getLecture().then((lecture) => {
```
Comment (amuamushu) [inline src/main/webapp/view/view.js:22]: Do you think you can switch to only using await or only using .then()? I think it's bad practice to mix them
.
https://docs.google.com/document/d/1J1s2uxmLxeuS6dEj110weXz1ajhAJRhUk0C7fowrNrc/edit

File: src/test/java/com/googleinterns/zoomtube/servlets/LectureServletTest.java [hunk 4]
```diff
@@ -67,6 +67,7 @@
   private PrintWriter writer;
 
   private static final String LINK_INPUT = "link-input";
```
Comment (amuamushu) [inline src/test/java/com/googleinterns/zoomtube/servlets/LectureServletTest.java:69]: should link-input be camel case?

Comment (jaysaleh) [reply src/test/java/com/googleinterns/zoomtube/servlets/LectureServletTest.java:69]: link-input is a the input name in the HTML

The style guide didn't include conventions for naming inputs so I defaulted to the convention for ID and class naming.
https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters

File: src/main/webapp/view/view.js [hunk 5]
```diff
@@ -14,12 +14,17 @@
...
 const PARAM_ID = 'id';
 const PARAM_VIDEO_ID = 'video-id';
+const ENDPOINT_LECTURE = '/lecture';
 
 /* exported LECTURE_ID */
 window.LECTURE_ID = getLectureId();
 window.VIDEO_ID = getVideoId();
 
-initialize();
+// TODO: Ask if calling .then() on async is bad practice.
```
Comment (f-camp) [inline src/main/webapp/view/view.js:23]: TODOs should be actionable code changes, this seems like something that needs to be done before the review.

**Comment by f-camp** [comment]
This doesn't look like we're reviewing for merging to the ```master``` branch. We should be doing reviews for the ```master``` branch merge only otherwise it seems we might need to re-review this when its time to merge.

File: src/main/webapp/view/view.js (hunk 5 above)
Comment (jaysaleh) [reply src/main/webapp/view/view.js]: go/js-practices/async-functions top of page three does something similar

**Pull request merged**